doc/architecture/timestamps_overview.adoc

```diff
@@ -0,0 +1,67 @@
+ifndef::include-only-once[]
+:root-path: ../
+include::{root-path}_config.adoc[]
+endif::[]
+= Timestamps
+
+== Simulation time
+
+`Simulation time` refers to the timestamp within a simulation environment.
+
+**Definition of zero time of simulation time**
+- Zero time is arbitrary but must be identical for all frames and corresponding messages (probably within one simulation).
+- Zero time does not need to coincide with the unix epoch.
+- It is recommended to use the start time of the simulation as zero time.
+
+**SensorData-specific timestamp**
+- Point in time that the sensor data message becomes available to the rest of the system (i.e. the driving functions).
+- Corresponds with the sending time / publish time.
```

Comment:
Do we have the terms sending time / publish time somewhere in the documentation? Otherwise we could remove this line.

Reply:
Yes, sending time is used in the class reference documentation. Carlo suggested to replace it with publish time but there was no final decision on that yet, so I added both as options as this is still a draft PR.